great_expectations/datasource/data_connector/inferred_asset_sql_data_connector.py

```diff
@@ -21,17 +21,17 @@ def __init__(
         name: str,
         datasource_name: str,
         execution_engine: Optional[ExecutionEngine] = None,
-        data_asset_name_prefix: str = "",
-        data_asset_name_suffix: str = "",
-        include_schema_name: bool = False,
-        splitter_method: str = None,
-        splitter_kwargs: dict = None,
-        sampling_method: str = None,
-        sampling_kwargs: dict = None,
-        excluded_tables: List = None,
-        included_tables: List = None,
-        skip_inapplicable_tables: bool = True,
-        introspection_directives: Dict = None,
+        data_asset_name_prefix: Optional[str] = "",
+        data_asset_name_suffix: Optional[str] = "",
+        include_schema_name: Optional[bool] = False,
+        splitter_method: Optional[str] = None,
+        splitter_kwargs: Optional[dict] = None,
+        sampling_method: Optional[str] = None,
+        sampling_kwargs: Optional[dict] = None,
+        excluded_tables: Optional[list] = None,
```

Comment:
Interesting! I just learned something new. I was wondering about the change from `List` to `list`. Looks like `list`, `tuple`, etc are now generic types & support specifying contents and are preferred over using `typing.List` which is deprecated and will be removed. https://stackoverflow.com/a/62033243/2916375, https://www.python.org/dev/peps/pep-0585/